style.css

```diff
@@ -3419,8 +3417,8 @@ p > video {
 	.archive article.type-newsletter .entry-summary,
 	.archive article.type-newsletter .entry-footer,
 	.archive article.type-newsletter .post-thumbnail {
-		margin-left: 8%;
-		margin-right: 8%;
+		margin-left: 6%;
+		margin-right: 6%;
 	}
```

Comment:
Now that this is equal to `type-newsletter_article`, let's move it to the same block (here and other places).